doc/en/pools.rst

```diff
@@ -331,45 +331,72 @@ in modules that matches the name_pattern under current working directory.
 
 .. code-block:: python
 
-    @task_target()
-    def make_multitest1():
-        # a test target shall only return 1 runnable object
-        test = MultiTest(name="Proj2-Suite1", suites=[Suite1()])
+    @task_target
+    def make_multitest():
+        # a test target only returns 1 runnable object
+        test = MultiTest(name="MTest", suites=[Suite()])
         return test
 
 Once found, task object will be created from the target, and scheduled to pool.
-It is also possible to create multiple task objects out of one target:
+Moreover, arguments for tasks can be specified in `schedule_all`, it is useful
+when many targets of the same kind share the same arguments.
+
+.. code-block:: python
+
+    plan.schedule_all(
```

Comment:
Here, mixing make_multitest() - schedule_all() - make_multitest(), makes the documentation confusing.

Reply:
I'm not very understanding it .